### DISCUSSION TITLE
Install solaris_exporter without internet

### DISCUSSION THREAD

**Discussion by rareman99** [n27051538/solaris_exporter#13] [Ideas]
Hi,
Can we install solaris_exporter without internet? Do it similar Solaris Repository.

**Comment by n27051538** [comment]
Hello,
You need to do offline installation on required Python modules.
Here you could find info about this
https://packaging.python.org/en/latest/tutorials/installing-packages/

**Reply by rareman99** [reply]
Thanks 👍
